packages/rsocket-core/src/ClientServerMultiplexerDemultiplexer.ts

```diff
@@ -318,14 +360,99 @@ class ConnectionFrameHandler implements FrameHandler {
       // TODO: throw an exception and close connection
     }
   }
+
+  close(error?: Error) {
+    this.keepAliveHandler.close();
+    this.rsocket.close?.call(this.rsocket, error);
+  }
+}
+
+class KeepAliveHandler implements FrameHandler {
+  private keepAliveLastReceivedMillis = Date.now();
+  private activeTimeout: any;
```

Comment:
Can be `Number`.

Reply:
ahh. Left it as any since in Node it is `node.Timout` while in the browser it is number